jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.j3md

```diff
@@ -56,6 +56,13 @@ MaterialDef PBR Lighting {
         // Parallax/height map
         Texture2D ParallaxMap -LINEAR
 
+        // Specular-AA
+        Boolean UseSpecularAA : true
+        // screen space variance,Use the slider to set the strength of the geometric specular anti-aliasing effect between 0 and 1. Higher values produce a blurrier result with less aliasing.
+        Float Sigma
+        // clamping threshold,Use the slider to set a maximum value for the offset that HDRP subtracts from the smoothness value to reduce artifacts.
+        Float Kappa
+
```

Comment:
Sorry to bother you again with these small details, could you please rename Sigma and Kappa to something like SpecularAASigma SpecularAAKappa ?

Reply:
The related parameter names have been updated

Reply:
Thank you, can you do the same for the defines SPECULARAA_THRESHOLD SPECULARAA_SCREEN_SPACE_VARIANCE ? After that i think it is ready to merge.

Reply:
> Thank you, can you do the same for the defines SPECULARAA_THRESHOLD SPECULARAA_SCREEN_SPACE_VARIANCE ? After that i think it is ready to merge.

The related macro definitions have been updated